float64 x
float64 y
float64 conf_score

